perception/traffic_light_visualization/src/traffic_light_roi_visualizer/nodelet.cpp

```diff
@@ -127,13 +127,23 @@ void TrafficLightRoiVisualizerNodelet::imageRoiCallback(
 {
   cv_bridge::CvImagePtr cv_ptr;
   try {
-    cv_ptr = cv_bridge::toCvCopy(input_image_msg, input_image_msg->encoding);
+    cv_ptr = cv_bridge::toCvCopy(input_image_msg, sensor_msgs::image_encodings::RGB8);
```

Comment:
I think input_image_msg->encoding is better because it is not necessarily RGB8.

Reply:
@yukkysaito 
Thanks for the comment.
I changed this to RGB8 because the color of the box to be drawn is decided based on the RGB8  assumption. https://github.com/autowarefoundation/autoware.universe/blob/ba29fcd7992b86c6a134e2bc20955d1f75a6004b/perception/traffic_light_visualization/src/traffic_light_roi_visualizer/nodelet.cpp#L94-L99

Reply:
I think the code should be input_image_msg->encoding. :pray: 
If it has to be RGB to cause an error, how about using guard with `if`?

Reply:
@yukkysaito 
Could you tell me why it should be input_image_msg->encoding ?

Do you mean color decision should be like this?
```
 if (result.label.find("red") != std::string::npos) {
   if (input_image_msg->encoding == sensor_msgs::image_encodings::RGB8) {
     color = cv::Scalar{255, 0, 0}; 
   } else if (input_image_msg->encoding == sensor_msgs::image_encodings::BGR8) {
     color = cv::Scalar{0, 0, 255}; 
   } else { 
     (set red color for the other encoding)
   }
 } else if (result.label.find("yellow") != std::string::npos) { 
   if ...
...

Reply:
@miursh input_image_msg  is encoded with `input_image_msg->encoding`, so it should be `input_image_msg->encoding` in code even if it is RGB8. 
So what if the topic to subscribe to is BGR8?

sample code:
```
if (input_image_msg->encoding == sensor_msgs::image_encodings::RGB8)
  throw exception;
cv_ptr = cv_bridge::toCvCopy(input_image_msg, sensor_msgs::image_encodings::RGB8);
```